README.md

```diff
@@ -2,33 +2,34 @@
 
 The Conctr library allows you to easily integrate your agent and device code with the [Conctr IoT Platform](https://conctr.com). This library provides an easy way to send data to a Conctr application. 
 
-Click [here](https://api.staging.conctr.com/docs) for the full documentation of the API.
+Click [here](https://api.staging.conctr.com/docs) for the full documentation of the Conctr API.
 
 ### Setup
 
-To use this library you will need to :
-- Register an account on the Conctr platform.
+To use this library you will need to:
+- [Register](https://staging.conctr.com/signup) for an account on the Conctr platform.
 - Create an application.
 - Create a model within the application.
 
-**To add this library to your project, add** `#require "conctr.agent.class.nut:1.0.0"` **to the top of your agent code and add** `#require "conctr.device.class.nut:1.0.0"` **to the top of your device code.**
+**To add this library to your project, add** `#require "conctr.agent.class.nut:1.0.0"` **to the top of your agent code and add** `#require "conctr.device.class.nut:1.0.0"` **to the top of your device code**
 
 ## Agent Class Usage
 
-### Constructor: Conctr(*appId, apiKey, model[, useAgentId][, region][, environment]*)
-The constructor takes three required parameters: your application id, API key and model. There is also  three optional parameter, the region to be used (defaults to us-west-2), the environment (defaults to staging) and the useAgentId (defaults to false).
+### Constructor: Conctr(*appId, apiKey, model[, options]*)
+
+The constructor takes three required parameters: your application ID, API key and model. These details can be found by navigating into your application on the Conctr platform, selecting on the *models* tab in the left side menu then click on the example button under the model you wish to use and chose the tab marked *Squirell*. There are also three optional parameters: the region to be used (defaults to `"us-west-2"`), the environment (defaults to `"staging"`) and the *useAgentId* (defaults to `false`) which can be passed in within a table as the options parameter.
```

Comment:
"*models* tab" should be "*Models* tab" and "example button" should be "*Example* button".
"*Squirell*" should be "*Squirrel*". "options" should be "*Options*".